src/shared/defaultAwsContext.ts

```diff
@@ -109,7 +134,11 @@ export class DefaultAwsContext implements AwsContext {
             }
         })
         await this.context.globalState.update(regionSettingKey, this.explorerRegions)
-        this._onDidChangeContext.fire(new ContextChangeEventsArgs(this.profileName, this.explorerRegions))
+        this._onDidChangeContext.fire(new ContextChangeEventsArgs(
```

Comment:
Generating the new ContextChangeEventsArgs object is repetitive. Can it be centralized?